docs/pr_review_process.md

```diff
@@ -0,0 +1,71 @@
+# Pull Request Review Process
+
+### Pull Request Author Rules
+
+1. PRs (Pull Requests) should be modular and as targeted as possible.
+2. PRs for new features should focus on a minimum viable product. Embellishments can be added in additional PRs.
+3. PRs should not contain unrelated changes as much as possible. Open a new PR for unrelated changes.
+4. Commits for PRs should be succinct but descriptive. Try to avoid trivial “update file.py” type commit messages.
+5. When a PR is ready for review, apply the “ready to review” label. At this point, additional changes should only be made if necessary or as part of the review. Let the team know it’s ready so someone can volunteer to review it.
+
+### Overview
+
+The steps below do not necessarily need to be followed precisely in this order. Not all PRs will require all steps and some will not require an in-depth review at all (e.g., minor documentation updates).
+
+### Step 0: Familiarize yourself with the PR
+
+Before starting the review it is convenient to understand the type of update and/or the intended functionality.  
+
+1. Inspect the files changed. 
+2. Read the updated documentation. To compile the documentation locally, run: `mkdocs serve --theme readthedocs` in a local terminal with an activated PlantCV environment (Testing the documentation is done in step 4). 
+
+### Step 1: Prepare materials for the review
+
+1. Maintain a shared team directory for reviews (data and notebooks) in the PlantCV Google Drive.
```

Comment:
Email plantcv@danforthcenter.org from a gmail account to have the team share access to this google drive.